dnf.spec

```diff
@@ -137,7 +137,9 @@ Requires:       python2-gpg
 BuildRequires:  pyliblzma
 Requires:       pyliblzma
 Requires:       %{name}-conf = %{version}-%{release}
+%if ! 0%{?rhel} >= 8
```

Comment:
Shouldn't this simply be
%if ! 0%{?rhel}

?
What you have works, but it's forward looking and no RHEL release ships deltarpm.  In the event that dnf were to be added to e.g. rhel7, you'd need to change it there.

Just a simple question/comment.  Do with it what you will.